app/Providers/AppServiceProvider.php

```diff
@@ -23,6 +23,8 @@ public function boot()
      */
     public function register()
     {
-        //
+        if ($this->app->environment('development')) {
```

Comment:
Scrut is complaining about this, should it not be if ($this->app->environment() === 'development')

?